tools/make/depcheck/src/allowlist.rs

```diff
@@ -199,13 +210,19 @@ pub const EXTRA_LOGGING_DEPS: &[&str] = &["cfg-if", "log"];
 /// This should rarely change, and if it does consider toggling features until it doesn't
 pub const EXTRA_ZIP_DEPS: &[&str] = &[
     "adler2",
+    "bumpalo",
     "byteorder",
     "crc32fast",
     "flate2",
+    "lockfree-object-pool",
     "miniz_oxide",
+    "once_cell",
     "ordered-float",
+    "serde-spanned",
     "serde-value",
+    "simd-adler32",
     "zip",
+    "zopfli",
 ];
```

Comment:
Did you do the "consider toggling features until it doesn't" ?

I remember that `zip` has tons and tons of features, most of which we don't need

Reply:
I did, yes. `deflate-flate2` doesn't work on its own, the only way to enable the rust backend is the full `deflate` feature, which also pulls in zopfli